sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -143,6 +145,9 @@ private[sql] class SQLConf extends Serializable with CatalystConf {
   private[spark] def parquetUseDataSourceApi =
     getConf(PARQUET_USE_DATA_SOURCE_API, "true").toBoolean
 
+  private[spark] def orcFilterPushDown =
+    getConf(ORC_FILTER_PUSHDOWN_ENABLED, "false").toBoolean
```

Comment:
Filter push-down is a pretty error prone feature, 'cause it requires different components to interoperate seamlessly. We'd rather disable it by default until it's proven stable.